Systems biology approaches for understanding SARS-CoV-2 pathogenesis

rays1987 opened the pull request:
Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@10.1038/s41576-020-0212-5]. Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@10.1002/prca.201300074].

The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@10.1038/s41586-020-2008-3; @10.1016/S0140-6736(20)30251-8; @10.1038/s41586-020-2012-7]. Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@10.1016/S0140-6736(20)30251-8]. However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@10.1016/S0140-6736(20)30251-8; @10.1038/s41586-020-2012-7]. 

The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.  In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@10.1016/j.chom.2020.03.002]. Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@10.1016/S2213-2600(20)30076-X]. 

Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently [@10.1126/science.abb2507]. Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.

There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@10.1016/j.cell.2020.02.052]. Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@10.1073/pnas.0407992101; @10.1016/j.bbrc.2004.02.074]. Considering the paucity of omics-level big data sets on for SARS-CoV-2 till now, the existing data hubs that contain adequate information for the other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), The Virus Pathogen Resource (ViPR) could be the useful resources for computational and bioinformatics research on SARS-CoV-2.

agitter commented on content/07.pathogenesis.md:
```diff
@@ -26,3 +26,22 @@ This section would also be great for the introduction of zoonotic diseases which
 
 [Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. 
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases (Hood and Tian, 2012, PMID: 23084773). 
```
The next step will be adding in the Manubot-style automatic references.  For example, this manuscript has a DOI, which you can get from PubMed or the journal website
![image](https://user-images.githubusercontent.com/3532898/77592593-7c83a580-6ec0-11ea-8870-d2fe1e773a90.png)

The DOI is the best identifier to use if the article has one.  You can replace the citation with the Manubot DOI citation syntax:
```suggestion
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004]. 
```

Now Manubot will automatically pull in the reference details!

rays1987 replied:
Many thanks for this suggestion. Please could you suggest how do it when there are more than one references? Are those need to be separated by , or ;

rays1987 replied:
I have resolved the issue. Please have a look.

cgreene commented on content/07.pathogenesis.md:
```diff
@@ -26,3 +26,22 @@ This section would also be great for the introduction of zoonotic diseases which
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
 [Sandipan Ray volunteered to lead this section]
```
```suggestion
```

cgreene commented on content/07.pathogenesis.md:
```diff
@@ -26,3 +26,22 @@ This section would also be great for the introduction of zoonotic diseases which
 [Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. 
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004]. 
+Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases (Eckhardt et al., 2020, PMID: 32060427). 
```
```suggestion
Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5]. 
```

rays1987 replied:
@cgreene
Citation issue resolved
Please review "Systems biology approaches for understanding SARS-CoV-2 pathogenesis  #68"

cgreene replied:
I see that the one that @agitter suggested was resolved and I suggested another one. There are a lot of references that don't appear to use the manubot style though. Can you take a screenshot of what you're seeing as you're resolving them?

rays1987 replied:
Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@10.1038/s41576-020-0212-5]. Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@10.1002/prca.201300074].
The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@10.1038/s41586-020-2008-3; @10.1016/S0140-6736(20)30251-8; @10.1038/s41586-020-2012-7]. Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@10.1016/S0140-6736(20)30251-8]. However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@10.1016/S0140-6736(20)30251-8; @10.1038/s41586-020-2012-7].
The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@10.1016/j.chom.2020.03.002]. Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@10.1016/S2213-2600(20)30076-X].
Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently [@10.1126/science.abb2507]. Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@10.1016/j.cell.2020.02.052]. Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@10.1073/pnas.0407992101; @10.1016/j.bbrc.2004.02.074]. Considering the paucity of omics-level big data sets on for SARS-CoV-2 till now, the existing data hubs that contain adequate information for the other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), The Virus Pathogen Resource (ViPR) could be the useful resources for computational and bioinformatics research on SARS-CoV-2.

cgreene commented:
Hi @rays1987 - I just did one of these so that I could get some instructions. Let me know how these work for you.

First, click the files changed tab.
![Screen Shot 2020-03-26 at 8 08 38 AM](https://user-images.githubusercontent.com/542643/77645424-15ee9e00-6f39-11ea-8b92-1cea5e29cbc9.png)

Next, click the three dots / edit:
![Screen Shot 2020-03-26 at 8 08 52 AM](https://user-images.githubusercontent.com/542643/77645443-1c7d1580-6f39-11ea-95a6-5d8b3d8b4e48.png)

There you'll see the text of the document, including your proposed changes.

If you scroll until you find a citation, then I'd go ahead and search for what you currently have.
![Screen Shot 2020-03-26 at 8 07 19 AM](https://user-images.githubusercontent.com/542643/77645514-33236c80-6f39-11ea-9436-798b2097a207.png)

The first result on google appeared to be it via pubmed, which is perfect!
![Screen Shot 2020-03-26 at 8 07 33 AM](https://user-images.githubusercontent.com/542643/77645538-3d456b00-6f39-11ea-8258-8fd1cb71183a.png)

I clicked to pubmed and found the DOI at the bottom:
![Screen Shot 2020-03-26 at 8 07 42 AM](https://user-images.githubusercontent.com/542643/77645559-49312d00-6f39-11ea-90cd-4a116665c53b.png)

I then updated it to cite the DOI with the [@doi:...] syntax.
![Screen Shot 2020-03-26 at 8 07 51 AM](https://user-images.githubusercontent.com/542643/77645570-4df5e100-6f39-11ea-9dde-0cb1c656a7cf.png)

If there were multiple citations, I would have used [@doi:...; @doi:...]. If you'd prefer to use pubmed IDs you can. Manubot supports many different types of citations. The full set is here: https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations

Thanks!

rays1987 commented:
@cgreene<https://github.com/cgreene>

Citation issue resolved. I have included DOI for each citation.

Please review the updated PR "Systems biology approaches for understanding SARS-CoV-2 pathogenesis #68"

________________________________
From: Casey Greene <notifications@github.com>
Sent: Thursday, March 26, 2020 8:12 AM
To: greenelab/covid19-review <covid19-review@noreply.github.com>
Cc: Ray, Sandipan <Sandipan.Ray@pennmedicine.upenn.edu>; Mention <mention@noreply.github.com>
Subject: [External] Re: [greenelab/covid19-review] Systems biology approaches for understanding SARS-CoV-2 pathogenesis (#68)


Hi @rays1987<https://github.com/rays1987> - I just did one of these so that I could get some instructions. Let me know how these work for you.

First, click the files changed tab.
[Screen Shot 2020-03-26 at 8 08 38 AM]<https://user-images.githubusercontent.com/542643/77645424-15ee9e00-6f39-11ea-8b92-1cea5e29cbc9.png>

Next, click the three dots / edit:
[Screen Shot 2020-03-26 at 8 08 52 AM]<https://user-images.githubusercontent.com/542643/77645443-1c7d1580-6f39-11ea-95a6-5d8b3d8b4e48.png>

There you'll see the text of the document, including your proposed changes.

If you scroll until you find a citation, then I'd go ahead and search for what you currently have.
[Screen Shot 2020-03-26 at 8 07 19 AM]<https://user-images.githubusercontent.com/542643/77645514-33236c80-6f39-11ea-9436-798b2097a207.png>

The first result on google appeared to be it via pubmed, which is perfect!
[Screen Shot 2020-03-26 at 8 07 33 AM]<https://user-images.githubusercontent.com/542643/77645538-3d456b00-6f39-11ea-8258-8fd1cb71183a.png>

I clicked to pubmed and found the DOI at the bottom:
[Screen Shot 2020-03-26 at 8 07 42 AM]<https://user-images.githubusercontent.com/542643/77645559-49312d00-6f39-11ea-90cd-4a116665c53b.png>

I then updated it to cite the DOI with the [@doi<https://github.com/doi>:...] syntax.
[Screen Shot 2020-03-26 at 8 07 51 AM]<https://user-images.githubusercontent.com/542643/77645570-4df5e100-6f39-11ea-9dde-0cb1c656a7cf.png>

If there were multiple citations, I would have used [@doi<https://github.com/doi>:...; @doi<https://github.com/doi>:...]. If you'd prefer to use pubmed IDs you can. Manubot supports many different types of citations. The full set is here: https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations

Thanks!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/greenelab/covid19-review/pull/68#issuecomment-604395049>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5KTRIJ4F4IPUIM26D6JADRJNBC3ANCNFSM4LTZVC4Q>.

rays1987 commented:
[cid:2fe5d1b3-3d36-4379-8e5e-1da17bc82e6b]
________________________________
From: Casey Greene <notifications@github.com>
Sent: Thursday, March 26, 2020 8:58 AM
To: greenelab/covid19-review <covid19-review@noreply.github.com>
Cc: Ray, Sandipan <Sandipan.Ray@pennmedicine.upenn.edu>; Mention <mention@noreply.github.com>
Subject: [External] Re: [greenelab/covid19-review] Systems biology approaches for understanding SARS-CoV-2 pathogenesis (#68)


@cgreene commented on this pull request.

________________________________

In content/07.pathogenesis.md<https://github.com/greenelab/covid19-review/pull/68#discussion_r398551111>:

> @@ -26,3 +26,22 @@ This section would also be great for the introduction of zoonotic diseases which
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis

 [Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases (Eckhardt et al., 2020, PMID: 32060427).


I see that the one that @agitter<https://github.com/agitter> suggested was resolved and I suggested another one. There are a lot of references that don't appear to use the manubot style though. Can you take a screenshot of what you're seeing as you're resolving them?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/greenelab/covid19-review/pull/68#discussion_r398551111>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5KTRIWCBJVSKV5U6LD2K3RJNGP7ANCNFSM4LTZVC4Q>.

agitter commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,12 @@ This section would also be great for the introduction of zoonotic diseases which
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last  ...
```
```suggestion
Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
```

Please use a newline "Enter key" after each sentence.  That makes it easier for experts to give comments on each sentence and for us to track revisions.  This paragraph would be converted to what I have above.

The references look better but need to have the `doi:` prefix.  I added them in this paragraph as an example.

rays1987 commented:
@agitter
Many thanks for the suggestions. Updated as suggested. Please review.
________________________________
From: Anthony Gitter <notifications@github.com>
Sent: Thursday, March 26, 2020 9:41 AM
To: greenelab/covid19-review <covid19-review@noreply.github.com>
Cc: Ray, Sandipan <Sandipan.Ray@pennmedicine.upenn.edu>; Mention <mention@noreply.github.com>
Subject: [External] Re: [greenelab/covid19-review] Systems biology approaches for understanding SARS-CoV-2 pathogenesis (#68)


@agitter commented on this pull request.

________________________________

In content/07.pathogenesis.md<https://github.com/greenelab/covid19-review/pull/68#discussion_r398579767>:

> @@ -25,4 +25,12 @@ This section would also be great for the introduction of zoonotic diseases which



 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis



-[Sandipan Ray volunteered to lead this section]

+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@10.1038/s41576-020-0212-5]. Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@10.1002/prca.201300074].



⬇️ Suggested change

-Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@10.1016/j.gpb.2012.08.004]. Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@10.1038/s41576-020-0212-5]. Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@10.1002/prca.201300074].

+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.

+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].

+Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].

+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].



Please use a newline "Enter key" after each sentence. That makes it easier for experts to give comments on each sentence and for us to track revisions. This paragraph would be converted to what I have above.

The references look better but need to have the doi: prefix. I added them in this paragraph as an example.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/greenelab/covid19-review/pull/68#pullrequestreview-382017905>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5KTRP5V476L4BLLDLR2VLRJNLQNANCNFSM4LTZVC4Q>.

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
```
```suggestion
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 
```
Two suggestions beyond just some minor flow suggestions:
1. I believe SARS-CoV-2 is the more up-to-date name (per https://www.nature.com/articles/s41564-020-0695-z)
2. These citations are causing a bug, but I think it's just because DOIs that have "(" or ")" sadly cause issues. For future reference, you can convert them to remove these characters at this link: http://shortdoi.org/. I believe it should work with the change below.
```suggestion
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10.1016/S0140-6736(20)30251-8]. 
```
Same DOI as above
```suggestion
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10.1016/S0140-6736(20)30251-8]. 
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7].
```
```suggestion
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
```
```suggestion
In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
+Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10.1016/S2213-2600(20)30076-X].
```
```suggestion
Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10/ggk6c5].
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
+Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10.1016/S2213-2600(20)30076-X].
+
+Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. 
```
```suggestion
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10.1016/S2213-2600(20)30076-X].
+
+Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. 
+Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently [@doi:10.1126/science.abb2507]. 
```
```suggestion
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently [@doi:10.1126/science.abb2507]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+
+Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. 
+Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently [@doi:10.1126/science.abb2507]. 
+Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. 
```
```suggestion
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. 
+The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
+
+There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 
```
```suggestion
There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
+
+There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 
+Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074]. 
```
```suggestion
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074]. 
```

rando2 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+
+There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 
+Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074]. 
+Considering the paucity of omics-level big data sets on for SARS-CoV-2 till now, the existing data hubs that contain adequate information for the other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), The Virus Pathogen Resource (ViPR) could be the useful resources for computational and bioinformatics research on SARS-CoV-2.
```
```suggestion
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
```

nilswellhausen commented:
Nicely written! In the part with the cryo-EM structure of the spike protein, I suggest making a reference to the neutralizing antibody that has been developed against that structure to inhibit viral infection (part of the treatment section). Also: I think a section might need to be updated now because this paper came out @rays1987 https://doi.org/10.1101/2020.03.24.004655. I can add that once it gets merged.

agitter commented:
> I can't confirm the content, but I think it looks good to move ahead!

@rando2 how would you like to handle content reviews?  Should we have a Manubot/grammar pull request approval and also a content/domain expert approval for all major contributions?

cgreene commented:
@rishirajgoel I saw your introduction over in #17. It looks like your expertise would really be ideal for taking a look at this. Can you take a look at the text that's changing here (click "Files Changed" tab) and mark any concerns you have or changes you would suggest using the plus mark that shows up when you mouse over the line number area?

![Screen Shot 2020-03-26 at 12 56 57 PM](https://user-images.githubusercontent.com/542643/77674001-5104c780-6f61-11ea-8408-f8df3ca6f5ba.png)

rishirajgoel commented:
> @rishirajgoel I saw your introduction over in #17. It looks like your expertise would really be ideal for taking a look at this. Can you take a look at the text that's changing here (click "Files Changed" tab) and mark any concerns you have or changes you would suggest using the plus mark that shows up when you mouse over the line number area?
> 
> ![Screen Shot 2020-03-26 at 12 56 57 PM](https://user-images.githubusercontent.com/542643/77674001-5104c780-6f61-11ea-8408-f8df3ca6f5ba.png)

Will do!

KhiabanianLab commented:
I will write a paragraph summarizing this pre-print later this afternoon: https://www.biorxiv.org/content/10.1101/2020.02.10.942748v1. Unfortunately I am new to GitHub, so hopefully I won't mess up things.

cgreene commented:
@KhiabanianLab - can you do that by first creating a new issue for the paper and then filing a PR to integrate it where it fts? https://github.com/greenelab/covid19-review/issues/new/choose

rishirajgoel commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
```
There was an interesting paper published yesterday showing the induction of neutralizing antibodies against the viral spike protein receptor-binding domain in humans post SARS-COV-2 infection: https://www.biorxiv.org/content/10.1101/2020.03.21.990770v1
An interesting paper that certainly merits discussion when thinking about epitope-specific responses. I can spend some time trying to identify similarities between the predicted epitopes and observed antibodies.

gregszetoAI replied:
To add to this, nice report on the breadth and kinetics of immune cells mobilized in response to SARS-CoV2 infection, suggesting a concerted T cell and B cell response is needed for effective viral clearance: https://www.nature.com/articles/s41591-020-0819-2

nilswellhausen replied:
@rishirajgoel: I am currently writing on a neutralizing antibody section in the treatments section. You are more then welcome to participate in that!

rishirajgoel commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
+Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10/ggk6c5].
```
The Lancet Respiratory Med paper certainly shows hyperactivation of CD4+ and CD8+ cells (incl high expression of cytotoxic granule proteins in the CD8+s), but their overall count was reduced. The authors specifically note an expansion of the Th17 expansion among CD4+ cells and speculate that these cells may promote tissue damage. There is some evidence to support these findings in other viral infections (I can provide citations) and so this is an interesting hypothesis).

Semantic note: the overactivation of T cells is not "caused" by an increase in Th17 and cytotoxic CD8+s. Rather, these findings indicate an activated T cell phenotype. 

Another group published a study looking at T cell phenotype here: https://www.nature.com/articles/s41423-020-0401-3

My summary of the major findings --> increased levels of HLA-DR (MHC-II) and TIGIT in CD8+s. No differences in activation markers on CD4+. Interestingly, HLA-DR/TIGIT on CD8+ were observed at higher levels in the severe vs mild COVID group, providing some more evidence that CD8+ activation and may be important in pathology. To me it's a bit unclear what is happening w/ the CD4+ cells...

rishirajgoel replied:
I should add that the Lancet Resp Med paper only phenotyped 1 patient and therefore should be considered with some caution. The cellular and molecular immuno paper I linked followed a larger cohort and reported similar findings in the CD8+ compartment so I think that's probably a solid angle to pursue.

rays1987 replied:
Yes, the Xu et al. Lancet Respir Med. 2020 is a case report from Wuhan, China (Treatment and Research Center for Infectious Diseases). Considering a single patient report (which is the same for many other top-tier publications on 2019-nCoV at this stage) the phenomenon is stated as "could be". The findings from this paper are highly interesting. Anyway, please feel free to include needful modifications.

rays1987 replied:
Many thanks for your valuable input.

rishirajgoel replied:
Agreed! And my pleasure. Lots of interesting things to think about. If you think the Cellular/Molecular Immuno paper merits inclusion in this section I can add a brief comment in the main text.

gregszetoAI replied:
Great discussion. I would add a few other potential references of interest including:

https://www.medrxiv.org/content/10.1101/2020.02.23.20026690v1
- single-cell seq of BAL also backs a dysregulated CD8 angle
- limited sample-size, but interesting to note potential sources of local cytokine storm in lungs
- caveat = all patients had pneumonia (including mild ones) which was surprising to me

https://www.medrxiv.org/content/10.1101/2020.02.18.20024364v1
- shows clear drops in T cell numbers
- implicates PD-1 and Tim3 in exhaustion as well
- inverse correlation between some circulating cytokines and T cell numbers

https://academic.oup.com/nsr/advance-article/doi/10.1093/nsr/nwaa041/5804736
- implicates Th1 subset in severe disease
- raises role for monocyte subpopulation as well

https://www.medrxiv.org/content/10.1101/2020.02.16.20023903v1
- decreased IFNg+ CD4+ T cells in severe patients
- further evidence of proteins involved in cytokine storm in severe COVID19

https://www.nature.com/articles/s41423-020-0402-2
- evidence of increased exhaustion of cytotoxic cells (CD8+ T cells and NK cells) in severe patients
- analysis of NKG2A exhaustion 

https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa248/5803306
- another subset analysis looking at CD4 and CD8 naive, helper, CD4 Tregs
- higher naive, and lower memory and Tregs in severe cases

gregszetoAI replied:
I would also encourage conservative interpretation of the Th17 finding in Lancet Resp Med. They show CCR6+ Th17s but don't use IL-17A staining to prove they are bonafide and functional Th17s. As shown in many other references, CCR6+ populations of CD4+ T cells can have functional diversity as noted in other viral infections: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995622/

cgreene replied:
This is really helpful. There's a lot of good stuff here. I'd propose this. I'm going to try to suggest a revision to be more conservative about Th17. Then, perhaps, we can get @rishirajgoel to come back through and suggest edits/revision to incorporate the additional papers mentioned.

At that point, perhaps @gregszetoAI and add the breadth of the literature that he raised. How does that sound?

rays1987 replied:
The statement on the findings from Xu et al. Lancet Respir Med. 2020 is tuned down as suggested. 
I think it might be a good idea to include further details in the "Immune Response to SARS-CoV-2" subsection under in the Pathogenesis section. 
Discussions on the papers suggested by @gregszetoAI and @rishirajgoel will be most suitable for that sub-section. What do you think @cgreene and HM Rando@rando2?

gregszetoAI replied:
Mostly agree. Given the systems bio section, I would think at least the single-cell seq paper may fit well here.

nilswellhausen replied:
I agree with @rays1987. I think the pathogenesis section needs some experts in virology and immunology and it seems like you guys have a good background in that. I outlined the pathogenesis section relatively early on just to spark some idea so feel free to adjust it to your needs but I think that understanding the basics of SARS-CoV-2 entry, replication and the immune response is obviously one the heart of COVID-19 biology. My suggestion would be that the systems part of this section should be more focused on the technologies and less on the biology. Most papers take advantage of systems approaches these days so it might also be worth thinking about splitting the systems bio part into where it fits into biology to avoid redundancy.

rays1987 commented:
Yes, the Xu et al. Lancet Respir Med. 2020 is a case report from Wuhan, China (Treatment and Research Center for Infectious Diseases). Considering a single patient report (which is the same for many other top-tier publications on 2019-nCoV at this stage) the phenomenon is stated as "could be". The findings from this paper are highly interesting. Anyway, please feel free to include needful modifications.

________________________________
From: rishirajgoel <notifications@github.com>
Sent: Thursday, March 26, 2020 1:34 PM
To: greenelab/covid19-review <covid19-review@noreply.github.com>
Cc: Ray, Sandipan <Sandipan.Ray@pennmedicine.upenn.edu>; Mention <mention@noreply.github.com>
Subject: [External] Re: [greenelab/covid19-review] Systems biology approaches for understanding SARS-CoV-2 pathogenesis (#68)


@rishirajgoel commented on this pull request.

________________________________

In content/07.pathogenesis.md<https://github.com/greenelab/covid19-review/pull/68#discussion_r398761044>:

> @@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which

 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis

-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
+A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates.
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43].
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002].
+Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10/ggk6c5].


I should add that the Lancet Resp Med paper only phenotyped 1 patient and therefore should be considered with some caution. The cellular and molecular immuno paper I linked followed a larger cohort and reported similar findings in the CD8+ compartment so I think that's probably a solid angle to pursue.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/greenelab/covid19-review/pull/68#discussion_r398761044>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5KTRKVQQQN2AMC4JL255DRJOGYVANCNFSM4LTZVC4Q>.

rays1987 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
```
Corrections seem good

rays1987 replied:
Referring to the individual sample types separately seems more precise

rays1987 commented on content/07.pathogenesis.md:
```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10.1016/S0140-6736(20)30251-8]. 
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin [@doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
```
It refers to the 3 genome sequence papers - One Lancet and Two Natures

rays1987 replied:
I have removed the immunology related discussion from this sub-section to avoid redundancy. We will add those and other suggested papers in the "Immune Response to SARS-CoV-2" sub-section.

cgreene replied:
Briefly corresponded with @rando2 . This was good to go for now & we will continue to revise as more is learned 👍
